### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by elenment**

### CODE REVIEW

**Comment by elenment** [comment]
My code runs without any issues when executing cargo b -r and it can deploy files normally. This is my program, and after my testing, there are no problems. However, when I execute cargo t -r, I encounter errors, similar to the situation when running the CI file, which prevents me from running the test code. Is there any way to solve this problem?

**Comment by elenment** [comment]
Program ID:0xee5df2bc579fd7ff48128415452eee8d3a242b6437bcd210870b83ed98e203c7

File: pebbles-game/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    }
+}
+
+// Handle function
+#[no_mangle]
+pub extern "C" fn handle() {
+    // Load user action
+    let action: PebblesAction = msg::load().expect("Unable to load PebblesAction");
+
+    unsafe {
```
Comment (MedovTimur) [inline pebbles-game/src/lib.rs:90]: it is not necessary to make the whole block unsafe, because it is only needed to take PEBBLES_GAME

**Comment by MedovTimur** [comment]
But on the whole, it's done right
Thanks for taking the course and understanding the material !

File: pebbles-game/src/lib.rs (hunk 1 above)
Comment (elenment) [reply pebbles-game/src/lib.rs:90]: I'm not very familiar with this aspect of knowledge, but after studying it, I will modify my code again.

**Pull request merged**